models/modelling/commissioning/projects/myria/int_myria_evaluation.sql

```
@@ -0,0 +1,174 @@
+, apc_encounter_summary as(
+   select
+        apc.sk_patient_id
+        ,m.hex_id
+        , apc.start_date as activity_date
+        , apc.end_date
+        , d.date as activity_date_range
+        , 'Inpatient' as activity_type
+        , case when apc.spec_comm_flag = 'Y' then 'spec_comm' else apc.admission_method_group end as activity_subtype
+        , sum(case when d.date = apc.start_date then 1 end) as encounters
+        -- spreads costs against all days in spell, if open spell, all days except last 7
+        , sum(cost / datediff(day, apc.start_date, coalesce(dateadd(dd, 1, apc.end_date), dateadd(dd, -6, current_date))) ) as cost 
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)